addons/account/models/account_invoice.py

```diff
@@ -1164,6 +1171,18 @@ class AccountInvoiceTax(models.Model):
     sequence = fields.Integer(help="Gives the sequence order when displaying a list of invoice tax.")
     company_id = fields.Many2one('res.company', string='Company', related='account_id.company_id', store=True, readonly=True)
     currency_id = fields.Many2one('res.currency', related='invoice_id.currency_id', store=True, readonly=True)
+    amount_in_group = fields.Monetary(compute='_compute_amount_in_group')
```

Comment:
Hey Quentin,

I know this really seems overkill, that's why I said it seemed weird in the original comment on the PR. As I said in another comment, the main reason is I needed a field for amount_in_group, but if I can achieve the same result without needing a field, I surely wouldn't do it that way as it would indeed be super simple, and would only required one method and no new field.

If you know a better approach to render the amount in the right way without this ugly hack, it would be great!